library/cwm/examples/example_helper.rb

```diff
@@ -0,0 +1,17 @@
+root_location = File.expand_path("../../../..", __FILE__)
```

Comment:
helper helps with loading lib, but problem is that Y2DIR is read at start of yast2 before reading helper